crates/aptos-rosetta/src/account.rs

```diff
@@ -170,28 +171,19 @@ async fn get_balances(
                                 currency: native_coin(),
                             })
                         }
-                    } else if account.is_operator_stake() {
-                        // For operator stake, filter on operator address
-                        let operator_address = account.operator_address()?;
-                        if let Some(contract) = store.staking_contracts.get(&operator_address) {
-                            if let Ok(response) = rest_client
-                                .get_account_resource_bcs::<StakePool>(
-                                    contract.pool_address,
-                                    "0x1::stake::StakePool",
-                                )
-                                .await
-                            {
-                                let stake_pool = response.into_inner();
-
-                                // Any stake pools that match, retrieve that.  Then update the total
-                                if let Ok(balance) =
-                                    get_stake_balance_from_stake_pool(&stake_pool, &account)
-                                {
-                                    balances.push(balance)
-                                }
-                            }
-                        }
-                    }
+                    } /* TODO: Right now operator stake is not supported
```

Comment:
Might as well just delete this code? We can resurrect it later by looking at history

Reply:
I don't really want to do that right now, since it'll be resurrected really soon, and I don't want to have to go back through the history for it.